### DISCUSSION TITLE
GTA V 1.0.3351.0_v2 compatibility

### DISCUSSION THREAD

**Discussion by DreadPirate09** [scripthookvdotnet/scripthookvdotnet#1511] [General]
Hey , just a question , the latest release should be compatible with v1.0.3351.0_v2 ? 
I saw that the last commit was an update for some enums versions and the last one was v1.0.3351.0 
Tried to replace the files in my GTAV folder but it is not logging anything so probably is not picked up.

**Comment by DreadPirate09** [comment]
Used the **ScriptHookVDotNet-v3.6.0-nightly.135** release

**Comment by kagikn** [comment]
First of all, I don't think there's the **official** game version v1.0.3351.0_v2 that is different from v1.0.3351.0? Also, SHVDN works without updating `GameVersion` enums, which is, updating them is just for convenience and not mandatory before SHVDN can try to work.

If you don't see `ScriptHookV.log` or `ScriptHookVDotNet.log`, you'll need to grant write permissions to the game folder for your Windows user.

**Comment by DreadPirate09** [comment]
I did grant write permissions to the game folder and it still not working . Not only the .log file is not showing up but the script I wrote in .cs is not logging as well (It suppose to write some logs in a file)
All I did is to add the files from **ScriptHookVDotNet-v3.6.0-nightly.135** except for the 2 folders and the README.txt in the game folder.
Am I missing something?

Thanks for the time!

**Reply by kagikn** [reply]
Can the SHVDN console be opened by pressing F4 or whatever you set to `ConsoleKeyBinding` in ScriptHookVDotNet.ini? Maybe you can see something informative. If not, you could see if asiloader.log is changed every time you start the game and what it says if it's the case.

**Reply by DreadPirate09** [reply]
I'll try in a bit and come back

**Reply by DreadPirate09** [reply]
It doesn't open and I can't find a asiloader.log file

**Reply by kagikn** [reply]
Can you see dinput8.dll and ScriptHookV.dll in the main folder, where your GTA5.exe is? Reinstall them from the archive file of Script Hook V again.

**Comment by DreadPirate09** [comment]
Damn , I totally forgot to install the file of script hook V xD . That was the problem . It works now. 

Thanks a lot , cheers!
